File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.js
function solution(n) {
    
    if(n==1) {
        return '수';
    }
    if(n==2) {
        return '수박';
    }
    
    const repeat = parseInt(n/2);
    let answer = ''
    
    for (let i=0; i<repeat; i++) {
        answer += '수박';
    }
    
    if (n%2==0) return answer;
    else return answer+'수';
    
    
}

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.py
def solution(n):
    
    if n==1:
        return '수'
    if n==2:
        return '수박'
    
    if n%2==0:
        answer = '수박'*(n//2)
    else:
        answer = '수박'*(n//2) + '수'
    
    return answer